scripts/bootstrap.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+from_source=""
+if [ "$1" = "-build" ]; then
+   from_source="t"
+fi
+
+set -e
+
+cd "$( dirname "$0" )/.."
+
+git fetch --tags --force --quiet
+version="$(git describe --tags "$(git rev-list --tags --max-count=1)")"
+os="$(uname)"
+arch="$(uname -m)"
+if [ "$from_source" = "" ]; then
+    if [ "$os" = "Darwin" ]; then
+	os="macos"
+    elif [ "$os" = "Linux" ]; then
+	os="linux"
+    else
+	echo "Unsupported OS."
+	exit 1
+    fi
+
+    curl -sLO "https://github.com/tigerbeetledb/tigerbeetle/releases/download/$version/tigerbeetle-$arch-$os-$version.zip"
+    unzip -qo "tigerbeetle-$arch-$os-$version.zip"
```

Comment:
Since this will overwrite any duplicates, is there any safety/sanity checking that we need to do, to verify that we're in the correct cwd before executing this step?

Reply:
It seems far-fetched, but is there any scenario where this might not err on the side of doing the safe thing?

Reply:
We'll also need to `chmod +x` here if using `unzip`.

Reply:
> Since this will overwrite any duplicates, is there any safety/sanity checking that we need to do, to verify that we're in the correct cwd before executing this step?

There's one `cd` in the script to make sure we're in the TigerBeetle root. So there shouldn't be anything more to check.

Reply:
> We'll also need to chmod +x here if using unzip.

Done!